### PR TITLE
Modify rule S2583: Migrate Python description to LaYC format

### PR DESCRIPTION
**Pull request by Swalkyn**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S2583/python/rule.adoc [hunk 1]
```diff
@@ -1,23 +1,69 @@
...
+    return "Result 2" # Never executed
 ----
 
+==== Compliant solution
+
+[source,python,diff-id=1,diff-type=compliant]
+----
+def foo(a, b):
+    if (a and not b):  # Noncompliant
+        doSomething()  # Never executed
```
Comment (anton-haubner-sonarsource) [inline rules/S2583/python/rule.adoc:51]: These comments probably got copied here by accident:
```suggestion
    if (a and not b):
        doSomething()
```

**Review by anton-haubner-sonarsource — CHANGES_REQUESTED** [review]
Looks good to me overall!
However, there is visual bug in SonarQube when rendering the unordered list that we probably should address.
I also added some other minor comments.

File: rules/S2583/python/rule.adoc [hunk 2]
```diff
@@ -1,23 +1,69 @@
+include::../description.adoc[]
+
```
[the remark this answers is not in the document]
Comment (Swalkyn) [reply rules/S2583/python/rule.adoc:2]: Indeed it does. I did not put it there initially, however the validator complained :shrug:

[the remark this answers is not in the document]
Comment (Swalkyn) [reply rules/S2583/python/rule.adoc:2]: I will remove the include for now as I could not find another solution.

**Pull request merged**